.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+Tips for a good bugreport
+=========================
+
+* Remember, if you are pedantically detailed about your problem, we can help you
+  better.
+* Please use markdown syntax, especially for Backtraces and config files!
+* Please provide a minimal working example. That means, you should cut out
+  irrelevant code from your muttrc or any other config file relevant to your
+  problem. Especially for complicated issues this is very valuable.
```

Comment:
I'd rephrase this as:

* Please provide a minimal working example. That means, you should try to replicate your issue with the minimal configuration from your muttrc and/or any other relevant config file. This is the details you should be posting.

the idea of a minimal working example is to help the user think about the problem and try to reduce it to its bare core.